marimo/_utils/variables.py

```diff
@@ -12,4 +12,4 @@ def is_mangled_local(name: str, cell_id: CellId_t) -> bool:
 
 
 def is_local(name: str) -> bool:
-    return name.startswith("_") and not name.startswith("__")
+    return name == "__" or (name.startswith("_") and not name.startswith("__"))
```

Comment:
`name.startswith("_") and not (name.startswith("__") and name.endswith("__"))` ?

Reply:
but  `__` satisfies `name.startswith("__") and name.endswith("__")`

Reply:
I think you probably want to just special case when every character is `_`

Reply:
I was contemplating how to simplify this logic as well, but it was just a special case!

Thanks for the review!